docs/core/tools/dotnet-pack.md

```diff
@@ -103,6 +103,15 @@ Defines the value for the `$(VersionSuffix)` MSBuild property in the project.
 
 Sets the verbosity level of the command. Allowed values are `q[uiet]`, `m[inimal]`, `n[ormal]`, `d[etailed]`, and `diag[nostic]`.
 
+> [!NOTE]
+> Web projects aren't packable by default. To override the default behavior, add the below property to your *.csproj* file:
+
+```xml
+<PropertyGroup>
+    <IsPackable>true</IsPackable>
+</PropertyGroup>
+```
```

Comment:
@mairaw will it be displayed nicely? It looks that the xml code block would appear out of the note block.

Reply:
You're right @pkulikov. I think we need to add > until the end of the note include the code sample piece

![image](https://user-images.githubusercontent.com/12971179/44288255-7d545000-a224-11e8-9a6c-6df007d2cddc.png)